tf_files/aws/modules/squid_nlb_central_csoc/README.md

```diff
@@ -0,0 +1,5 @@
+# Description
+A centralised NLB set-up for Squid proxy will be installed in the CSOC account. This is going to be a centralised squid set-up utilized by all the data commons VPC.
+
+   
+
```

Comment:
Let's make a lucid chart SVG, and link it here:
* CSOC
   - 3 subnets
   - autoscaling group
   - NLB 
   - endpoint service?  is that a thing?
* Child VPC
    - vpc endpoint
    - note about cloud-proxy.internal.io DNS entry

Also put a note about put a note about how we cannot point a default route at the endpoint ...

Reply:
Also put a note about pricing of bandwidth through the VPC endpoint